e2e/scripts/setup-jira-playwright.ts

```
@@ -0,0 +1,383 @@
+          console.log(`  [${elapsed}s] ✗ ${msg}`);
+          console.log('');
+          console.log('  === Docker logs (error detected) ===');
+          console.log(logs);
+          console.log('  =====================================');
+          return { ready: false, error: msg };
+        }
+      }
+
+      // Show latest log line
+      const lines = logs.split('\n').filter((l) => l.trim());
+      const latest = lines[lines.length - 1] || '';
```

ESLint

[error] Prefer `.at(…)` over `[….length - index]`.

(unicorn/prefer-at)